server/views/serviceProviderReferrals/interventionProgress.njk

```diff
@@ -0,0 +1,50 @@
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/tag/macro.njk" import govukTag %}
+
+{% extends "../partials/layout.njk" %}
+
+{% set pageTitle = "HMPPS Interventions" %}
+{% block pageTitle %}
+  {{ pageTitle }}
+  - GOV.UK
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-xl">{{ presenter.text.title }}</h1>
+
+      <h2 class="govuk-heading-m">Initial assessment appointment</h2>
+
+      <p class="govuk-body">
+      Complete the initial assessment within 10 working days from receiving a new referral. Once you enter the appointment details, you will be able to change them.
+      </p>
+
+      {{ govukSummaryList(initialAssessmentSummaryListArgs(govukTag)) }}
+
+      <h2 class="govuk-heading-m">Action plan</h2>
+
+      <p class="govuk-body">
+      Please complete the service user’s action plan and submit it to the probation practitioner for approval.
+      </p>
+
+      {{ govukSummaryList(actionPlanSummaryListArgs(govukTag, csrfToken)) }}
```

Comment:
personally i'm not a huge fan of passing these template vars back into the js but it's not a blocker for me

Reply:
Me neither, but especially in the case of the Nunjucks macro I think it would take a bigger refactor, because somehow we'd need to I guess import all of those `govuk` macros into our global Nunjucks environment and then pass this environment through to all of the views so that we can pull the macros out of it. Do you think it's worth investing time in that (or do you have an alternative in mind)?

Reply:
(The CSRF token is an easier one to fix, we could just pass that one in from the controller)